src/modules/coreHttpApi/openapi.yml

```diff
@@ -3508,6 +3524,10 @@ paths:
                 ephemeral:
                   description: If set to true the token will will not be cached in the database of the connector. Note that you will not be able to fetch this token unless you remember the truncatedReference of the token. Defaults to true. Will be ignored if Accept is set to image/png.
                   type: boolean
+                forIdentity:
+                  $ref: "#/components/schemas/Address"
+                  nullable: true
+                  description: The only Identity that may load this Token. If the RelationshipTemplate can only be loaded by one Identity, that Identity must also be given here.
```

Comment:
> If the RelationshipTemplate can only be loaded by one Identity

Do you mean, if `maxNumberOfAllocations` of the RelationshipTemplate is set to one? If so, please make that clearer here.

Reply:
I meant something different, thanks for pointing that out